crmsh/bootstrap.py

```diff
@@ -969,8 +962,11 @@ def ssh_copy_id_no_raise(local_user, remote_user, remote_node, shell: sh.LocalSh
     if utils.check_ssh_passwd_need(local_user, remote_user, remote_node, shell):
         configure_ssh_key(local_user)
         public_keys = ssh_key.fetch_public_key_file_list(None, local_user)
+        sleep(5)    # bsc#1243141: sshd PerSourcePenalties
```

Comment:
I'm not clear why sleep 5 seconds can avoid PerSourcePenalties?
Please also write the details into git commit log